### PR TITLE
Updated supported data formats supported by Tidb Lightning

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Updated supported data formats supported by Tidb Lightning

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/7176
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by glorv — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-faq.md [hunk 1]
```diff
@@ -65,9 +65,12 @@ ADMIN CHECKSUM TABLE `schema`.`table`;
...
 ```
 
-## What kind of data source format is supported by TiDB Lightning?
+## What kinds of data source formats are supported by TiDB Lightning?
 
-TiDB Lightning only supports the SQL dump generated by [Dumpling](/dumpling-overview.md) or [CSV files](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md) stored in the local file system.
+TiDB Lightning supports:
+
+- Importing CSV, [Dumpling](/dumpling-overview.md) or [Amazon Aurora Parquet](/migrate-from-aurora-using-lightning.md) output files.
+- Reading data from the local disk or [Amazon S3 cloud storage](/br/backup-and-restore-storages.md).
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-faq.md:73]:
```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-aurora-to-tidb.md).
- Reading data from a local disk or from the Amazon S3 storage. For details, see [External Storages](/br/backup-and-restore-storages.md).
```
这段话官网中英文都改过了，链接如下：
ZH：https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview
EN：https://docs.pingcap.com/tidb/stable/tidb-lightning-overview

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**